### PR TITLE
Tests for All Modules, M - P Excluding Misc

### PR DESCRIPTION
**Pull request by joshmgrant**
Tests have been added for

- `malapropisms.misc`
- `mixed_metaphors.misc`
- `mondegreens.misc`
- `needless_variants.misc`
- `nonwords.misc`
- `oxymorons.misc`
- `psychology.misc`

### CODE REVIEW

File: tests/test_mixed_metaphors.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            """Smoke phrase with nothing flagged.""") == []
+        assert chk.check_bottleneck(
+            """The project produced a huge bottleneck.""") != []
+
+    def test_smoke_misc(self):
+        """Basic smoke test for check_misc."""
+        assert chk.check_misc(
+            """Smoke phrase with nothing flagged.""") == []
+        assert chk.check_misc(
+            """Writing tests is not rocket surgery.""") != []
```
Comment (suchow) [inline tests/test_mixed_metaphors.py]: 😄

File: tests/test_psychology.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            """Smoke phrase with nothing flagged.""") == []
+        assert chk.check_p_equals_zero(
+            """The effect was highly signficant at p = 0.00.""") != []
+
+    def test_smoke_mental_telepathy(self):
+        """Basic smoke test for psychology.misc.check_mental_telepathy."""
+        assert chk.check_mental_telepathy(
+            """Smoke phrase with nothing flagged.""") == []
+        assert chk.check_mental_telepathy(
+            """I've been practicing mental telepathy.'""") != []
```
Comment (suchow) [inline tests/test_psychology.py]: I think there's an unintended single quotation mark after the period.

**Comment by suchow** [comment]
Thanks! Looks good. I made a line comment with a minor tweak, otherwise ready to merge.

File: tests/test_psychology.py (hunk 2 above)
Comment (joshmgrant) [inline tests/test_psychology.py]: Good catch, thanks! I'll update the resubmit the PR.

**Pull request merged**